scripts/place_maker.py

```diff
@@ -133,14 +148,26 @@ def set_tags(content, args):
                         dest='message', help='Commit message.')
     parser.add_argument('--owner', default='admin',
                         dest='owner', help='Content owner. Defaults to "admin"')
-    parser.add_argument('--groups', nargs='+', default=[],
-                        dest='groups', help='Group names. Separated by spaces.')
-    parser.add_argument('--creators', nargs='+', default=[],
-                        dest='creators', help='Creators. Separated by spaces.')
-    parser.add_argument('--contributors', default=[],
-                        dest='contributors', nargs='+', help='Contributors. Separated by spaces.')
-    parser.add_argument('--tags', default=[], dest='subjects', nargs='+',
-                        help='Tags (subjects). Separated by spaces.')
+    parser.add_argument('--groups',
+                        default=[],
+                        dest='groups', 
+                        type=lambda s: re.split(r' |,\s*', s),
```

Comment:
This is a fairly unusual use of `type`, but I guess it works as a means to coerce a string into a list with more complex formatting than would be provided by `nargs='+'`.